### ISSUE TITLE
Feature request/bug: hyperlinks are not parsed

### ISSUE THREAD

**Issue by waldekmastykarz** [StfBauer/markshell#10]
Hyperlinks don't seem to be parsed:

<img width="920" alt="image" src="https://user-images.githubusercontent.com/11164679/91336288-fd2d3180-e7d1-11ea-883e-ad08a918ede3.png">

Is it intentional? If so, would it be possible to have them displayed in shell like:

```
see documentation (https://someurl.com)
```

which is more human-friendly than raw MD?

**Comment by StfBauer** [comment]
Two flavours - Hyperlinks that start with `(.` will be deleted unless you don't want to always write the full URL in there. Nice to have to replace it with the full URL but it could be tricky.

Other links will be formatted like you suggested.

**Comment by waldekmastykarz** [comment]
Just to check, will the URL be deleted and the text kept?

**Comment by StfBauer** [comment]
Url deleted text kept to avoid ../ links that are not usefull

**Comment by waldekmastykarz** [comment]
Make sense. At some point I thought that perhaps it would be good to have a way to specify the base URL in options, but that might be excessive and an unnecessary complication.

**Comment by StfBauer** [comment]
So the text like this:
<img width="859" alt="image" src="https://user-images.githubusercontent.com/5503835/91647500-beaaa780-ea5b-11ea-8669-084590ddfd80.png">
Get's stripped like this:

**Comment by StfBauer** [comment]
So made it now like this.

### Case 1:
```markdown
This is a link [https://n8d.at](https://n8d.at) for markshell.
```
translate into:

```
This is a link https://n8d.at for markshell.
```

### Case 2:
```markdown
This is a link [N8D](https://n8d.at) for markshell.
```
translate into:

```
This is a link N8D (https://n8d.at) for markshell.
```

### Case 3:
```markdown
This is a link [N8D](../samle-file.md) for markshell.
```
translate into:

```
This is a link N8D for markshell.
```

Makes sense?

**Comment by waldekmastykarz** [comment]
Yes, it does

**Comment by StfBauer** [comment]
Fixed in 📦  markshell@0.0.6-beta3

**Comment by waldekmastykarz** [comment]
Confirmed fixed
